### ISSUE TITLE
{Linked:Bug13586410}[Adaptive Cards > Designer] - Button present in the response does not have discernible text.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9165]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 24H2 (OS Build 27766.1000)
Browser Version: Version 132.0.2186.2 (Official build) dev (64-bit)
Tool: Accessibility Insight For Web

**Pre-Requisite:**
Run the below attached JSON file under 'Copy Card Payload editor.'

**Repro Steps:**
1. Open the above URL on the Edge Dev browser.
2. Press tab key and navigate to the 'Book this trip' button.
3. Press enter key to activate the 'Book this trip' button.
4. Now run the tool and observe the issue.

**Actual Result:**
Button present in the response does not have discernible text.

NVDA is announcing as ‘Status bar button’.

**Expected Result:** 
Button present in the response should have discernible text.

**User Impact:**

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

<img width="963" alt="Image" src="https://github.com/user-attachments/assets/1d46a2c5-13d5-4a21-bad0-3b3646d0e509" />

![Image](https://github.com/user-attachments/assets/1108b955-d90e-4ccd-ab0e-425c99344565)

### Card JSON

```JSON
Vignesh Vijayaraghavan
commented Tuesday


@Vaibhav Gupta (HCL Technologies Corporate Services) @Arvind Prakash (HCL Technologies Corporate Services)  Any adaptive card with a submit button would do.
Following JSON is an example: 
{
      "type": "AdaptiveCard",
      "version": "1.5",
      "speak": "First trip. 24-7 Test Route AM, Apr 24. Depart from REDMOND TECHNOLOGY STATION (RTS) at 02:01 PM. Arrive at 10TH AVE E & E ALOHA ST at 02:15 PM. ",
      "body": [
        {
          "type": "Container",
          "items": [
            {
              "type": "ColumnSet",
              "columns": [
                {
                  "type": "Column",
                  "width": "stretch",
                  "items": [
                    {
                      "type": "ColumnSet",
                      "columns": [
                        {
                          "type": "Column",
                          "width": "auto",
                          "items": [
                            {
                              "type": "TextBlock",
                              "size": "medium",
                              "weight": "bolder",
                              "text": "24-7 Test Route AM, Apr 24",
                              "wrap": true,
                              "maxLines": 1,
                              "style": "heading",
                              "spacing": "small"
                            }
                          ]
                        }
                      ]
                    }
                  ]
                }
              ]
            },
            {
              "type": "TextBlock",
              "weight": "lighter",
              "text": "Depart from",
              "wrap": true,
              "maxLines": 1
            },
            {
              "type": "ColumnSet",
              "columns": [
                {
                  "type": "Column",
                  "width": "stretch",
                  "items": [
                    {
                      "type": "TextBlock",
                      "text": "REDMOND TECHNOLOGY STATION (RTS)",
                      "wrap": true,
                      "maxLines": 1,
                      "spacing": "small"
                    }
                  ]
                },
                {
                  "type": "Column",
                  "width": "auto",
                  "items": [
                    {
                      "type": "TextBlock",
                      "text": "02:01 PM",
                      "horizontalAlignment": "right",
                      "wrap": true,
                      "maxLines": 1,
                      "spacing": "small"
                    }
                  ]
                }
              ],
              "spacing": "small"
            },
            {
              "type": "TextBlock",
              "weight": "lighter",
              "text": "Arrive at",
              "wrap": true,
              "maxLines": 1
            },
            {
              "type": "ColumnSet",
              "columns": [
                {
                  "type": "Column",
                  "width": "stretch",
                  "items": [
                    {
                      "type": "TextBlock",
                      "text": "10TH AVE E & E ALOHA ST",
                      "wrap": true,
                      "maxLines": 1,
                      "spacing": "small"
                    }
                  ]
                },
                {
                  "type": "Column",
                  "width": "auto",
                  "items": [
                    {
                      "type": "TextBlock",
                      "text": "02:15 PM",
                      "horizontalAlignment": "right",
                      "wrap": true,
                      "maxLines": 1,
                      "spacing": "small"
                    }
                  ]
                }
              ],
              "spacing": "small"
            },
            {
              "type": "ColumnSet",
              "columns": [
                {
                  "type": "Column",
                  "width": "auto",
                  "items": [
                    {
                      "type": "ActionSet",
                      "actions": [
                        {
                          "type": "Action.Submit",
                          "data": {
                            "id": "30a0705b-c0a4-f29b-7ca3-e5c03f71583c"
                          },
                          "title": "Book this trip"
                        }
                      ]
                    }
                  ]
                }
              ]
            }
          ],
          "spacing": "medium",
          "separator": true
        }
      ]
    }
```

### Sample Code

Originally tracking with - 

https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/13586410

**Comment by sayali-MSFT** [comment]
@vagpt ,Thank you for your inquiry about your Teams app development issue!

We are checking the issue. We will get back to you shortly.

**Comment by SatwikKrSharma** [comment]
Please try with the below JSON
`{
      "type": "AdaptiveCard",
      "version": "1.5",
      "speak": "First trip. 24-7 Test Route AM, Apr 24. Depart from REDMOND TECHNOLOGY STATION (RTS) at 02:01 PM. Arrive at 10TH AVE E & E ALOHA ST at 02:15 PM. ",
      "body": [
        {
          "type": "Container",
          "items": [
            {
              "type": "ColumnSet",
              "columns": [
                {
                  "type": "Column",
                  "width": "stretch",
                  "items": [
                    {
                      "type": "ColumnSet",
                      "columns": [
                        {
                          "type": "Column",
                          "width": "auto",
                          "items": [
                            {
                              "type": "TextBlock",
                              "size": "medium",
                              "weight": "bolder",
                              "text": "24-7 Test Route AM, Apr 24",
                              "wrap": true,
                              "maxLines": 1,
                              "style": "heading",
                              "spacing": "small"
                            }
                          ]
                        }
                      ]
                    }
                  ]
                }
              ]
            },
            {
              "type": "TextBlock",
              "weight": "lighter",
              "text": "Depart from",
              "wrap": true,
              "maxLines": 1
            },
            {
              "type": "ColumnSet",
              "columns": [
                {
                  "type": "Column",
                  "width": "stretch",
                  "items": [
                    {
                      "type": "TextBlock",
                      "text": "REDMOND TECHNOLOGY STATION (RTS)",
                      "wrap": true,
                      "maxLines": 1,
                      "spacing": "small"
                    }
                  ]
                },
                {
                  "type": "Column",
                  "width": "auto",
                  "items": [
                    {
                      "type": "TextBlock",
                      "text": "02:01 PM",
                      "horizontalAlignment": "right",
                      "wrap": true,
                      "maxLines": 1,
                      "spacing": "small"
                    }
                  ]
                }
              ],
              "spacing": "small"
            },
            {
              "type": "TextBlock",
              "weight": "lighter",
              "text": "Arrive at",
              "wrap": true,
              "maxLines": 1
            },
            {
              "type": "ColumnSet",
              "columns": [
                {
                  "type": "Column",
                  "width": "stretch",
                  "items": [
                    {
                      "type": "TextBlock",
                      "text": "10TH AVE E & E ALOHA ST",
                      "wrap": true,
                      "maxLines": 1,
                      "spacing": "small"
                    }
                  ]
                },
                {
                  "type": "Column",
                  "width": "auto",
                  "items": [
                    {
                      "type": "TextBlock",
                      "text": "02:15 PM",
                      "horizontalAlignment": "right",
                      "wrap": true,
                      "maxLines": 1,
                      "spacing": "small"
                    }
                  ]
                }
              ],
              "spacing": "small"
            }
          ],
          "spacing": "medium",
          "separator": true
        }
      ], "actions": [
    {
      "type": "Action.Submit",
      "title": "Book this trip",
      "data": {
        "id": "30a0705b-c0a4-f29b-7ca3-e5c03f71583c"
      }
    }
  ]
    }`

**Comment by vagpt** [comment]
Hi @SatwikKrSharma,

Thanks for providing the JSON file.

Although using the attached JSON file we are not getting the status bar. Please refer the below snippet for reference.

![Image](https://github.com/user-attachments/assets/c05b5011-b2fa-4253-96cb-4694ab0dbfa7)

**Comment by SatwikKrSharma** [comment]
Hi @vagpt, I do not think the status bar is part of Adaptive Card JSON
